File: src/main/java/de/thatsich/intellie/common/module/item/AItemBlock.java
package de.thatsich.intellie.common.module.item;

import net.minecraft.init.Blocks;
import net.minecraft.item.ItemBlock;


public class AItemBlock extends ItemBlock implements IItemBlock
{
	public AItemBlock ( AItemInfo info, AItemConfig config, AItemRecipe recipe )
	{
		super( Blocks.dirt );
	}
}

File: src/main/scala/de/thatsich/minecraft/intellie/common/module/item/AItemBlock.scala
package de.thatsich.minecraft.intellie.common.module.item

import net.minecraft.item.ItemBlock
import de.thatsich.minecraft.intellie.common.module.{ARecipe, AConfig}
import net.minecraft.init.Blocks

/**
 *
 *
 * @author thatsIch
 * @since 07.04.2014.
 */
abstract class AItemBlock(info: AItemInfo, config: AConfig, recipe: ARecipe)
	extends ItemBlock(Blocks.dirt)
	with IItemBlock
{


}
